gdnative-sys/build.rs

```diff
@@ -91,29 +91,91 @@ mod header_binding {
             .to_string_lossy(),
         );
 
+        let mut android_ndk_root: Option<PathBuf> = None;
+
+        if Path::join(&android_sdk_root, "ndk").exists() {
```

Comment:
`Path::join(&android_sdk_root, "ndk")` is used again further down, could be extracted into a variable `sdk_path`.